warp/warp-configs.js

```diff
@@ -31,6 +31,20 @@ async function warp(network) {
     ));
     return warp
 
+  } else if (network === 'mainNet' && customDREURL) {
```

Comment:
This block of code should never run, because the 1st half of the condition is already met on line 8: `if (network === 'mainNet')`
and this block is on the `else if`

Reply:
Also, (not mandatory, just checking) is it ok to pass the `customDREURL` param without sanitizing or checking if it's at least a valid URL?

Reply:
Ahhhhh your right on the if block, don't worry this is only the server that the SDK interacts with - as the SDK is in TS it does all the checking :)